### PR TITLE
View Directive @notset

### PR DESCRIPTION
**Pull request by NathnC**
Since I use isset() quite a lot, with the new @isset directive I thought it'd be nice to have a single @notset one so it's easier to do the opposite without the need for a else statement

### CODE REVIEW

File: tests/View/Blade/BladeHelpersTest.php [hunk 1]
```diff
@@ -11,5 +11,6 @@ public function testEchosAreCompiled()
         $this->assertEquals('<?php dd($var1); ?>', $this->compiler->compileString('@dd($var1)'));
         $this->assertEquals('<?php dd($var1, $var2); ?>', $this->compiler->compileString('@dd($var1, $var2)'));
         $this->assertEquals('<?php dump($var1, $var2); ?>', $this->compiler->compileString('@dump($var1, $var2)'));
+        $this->assertEquals('<?php !isset($var1); ?>', $this->compiler->compileString('@notset($var1)'));
```
Comment (jmarcher) [inline tests/View/Blade/BladeHelpersTest.php]: Test is wrong... you need to run them to check if they work...

File: src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php [hunk 2]
```diff
@@ -151,6 +151,27 @@ protected function compileEndIsset()
     {
         return '<?php endif; ?>';
     }
+
```
Comment (jmarcher) [inline src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php]: Delete this tab

**Review by jmarcher — CHANGES_REQUESTED** [review]

**Comment by jmarcher** [comment]
I do not think this directive is needed. You should check if the tests pass before sending a PR

**Comment by taylorotwell** [comment]
No plans on adding.

**Pull request closed without merging**